src/components/IncorporationAgreement/AgreementType.vue

```diff
@@ -48,8 +48,12 @@
 import { Component, Vue, Prop } from 'vue-property-decorator'
 import { Action, Getter, State } from 'vuex-class'
 
-// Interfaces
+// Interfaces & enums
 import { ActionBindingIF, IncorporationAgreementTypeIF } from '@/interfaces'
+import { RouteNames } from '@/enums'
+
+// Modules
+import { CorpTypeCd, GetCorpFullDescription } from '@bcrs-shared-components/corp-type-module/index'
```

Comment:
I would prefer to see the method re-imported locally, like here:
https://github.com/bcgov/business-filings-ui/blob/1e77360722b40500baabe2e9e50b02b604ef864d/src/mixins/enum-mixin.ts#L140

Reply:
Added enum mixin and applied in next commit

Reply:
I'm not sure i'm a big fan of this. It seems to just add more code since we now need an enum to distribute the methods, although in cases outside components where we utilize these methods, they require module imports anywho. 
I think i'd prefer to leave these as is, thoughts?

Reply:
Updated but not my preference 😆